src/app/globals.css

```
@@ -44,20 +44,20 @@
 :root {
   --radius: 0.625rem;
   --background: oklch(1 0 0);
-  --foreground: oklch(0.145 0 0);
+  --foreground: #002147;
   --card: oklch(1 0 0);
-  --card-foreground: oklch(0.145 0 0);
+  --card-foreground:#002147;
   --popover: oklch(1 0 0);
-  --popover-foreground: oklch(0.145 0 0);
+  --popover-foreground:#002147;
   /*Primary-Color e.g button*/
```

Stylelint (17.5.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)